ctapipe/analysis/Sensitivity.py

```diff
@@ -0,0 +1,790 @@
+from itertools import chain
+import astropy.units as u
+from astropy.table import Table
+from scipy.optimize import minimize
+import scipy.integrate as integrate
+
+import numpy as np
+
+from matplotlib import pyplot as plt
+
+
+# tau, the superior circle number
+np.tau = 2*np.pi
```

Comment:
This was meant of course more as a joke than a serious fix of numpy.
I will change it to use `2*np.pi` inline.